doc/Language/quoting.pod6

```diff
@@ -174,38 +174,26 @@ My favorite color is blue!
 =end code
 
 X<|\ (quoting)>
+
 The C<qq> form – usually written using double quotes – allows for
-interpolation of backslash sequences and variables, i.e., variables can be
-written within the string so that the content of the variable is inserted into
-the string. It is also possible to escape variables within a C<qq>-quoted
-string:
+interpolation of backslash escape sequences (like C<q:backslash>), all sigiled
+variables (like C<q:scalar:array:hash:function>), and any code inside C<{...}>
+(like C<q:closure>).
 
-=begin code :allow<B> :preamble<my $color = 'blue'>
-say B<">The B<\>$color variable contains the value '$color'B<">;
-=end code
+=head3 Interpolating variables
 
-=begin code :lang<text>
-The $color variable contains the value 'blue'
-=end code
+Inside a C<qq>-quoted string, you can use variables with a sigil to trigger
+interpolation of the variable's value.  Variables with the C<$> sigil are
+interpolated whenever the occur (unless escaped); that's why, in the example
+above, C<"$color"> became C<blue>.
 
-Another feature of C<qq> is the ability to interpolate Raku code from
-within the string, using curly braces:
+Variables with other sigils, however, only trigger interpolation when you follow
+the variable with the appropriate postfix (C<[]> for Arrays, C«<>», for Hashes,
+C<&> for Subs). This allows you to write expressions like
+C<"documentation@raku.org"> without interpolating the C<@raku> variable.
 
-=begin code :allow<B L>
-my ($x, $y, $z) = 4, 3.5, 3;
-say "This room is B<$x> m by B<$y> m by B<$z> m.";
-say "Therefore its volume should be B<{ $x * $y * $z }> m³!";
-=end code
-
-=begin code :lang<text>
-This room is 4 m by 3.5 m by 3 m.
-Therefore its volume should be 42 m³!
-=end code
-
-By default, only variables with the C<$> sigil are interpolated normally.
-This way, when you write C<"documentation@raku.org">, you aren't
-interpolating the C<@raku> variable. If that's what you want to do, append
-a C<[]> to the variable name:
+To interpolate an Array (or other L<Positional|/type/Positional> variable),
```

Comment:
Probably just talk about anything with the `@` sigil instead. I really have no idea why this happens, BTW